default.nix

```
@@ -134,15 +134,15 @@ in rec {
           || throw "Thunk at ${toString p} has files in addition to ${name} and optionally default.nix and .attr-cache. Remove either ${name} or those other files to continue (check for leftover .git too)."
         else false;
     in
-      if isObeliskThunkWithThunkNix then import (p + /thunk.nix)
+      if isObeliskThunkWithThunkNix then import ("${p}/thunk.nix")
```

FYI
```suggestion
      if isObeliskThunkWithThunkNix then import (p + "/thunk.nix")
```
is also an option

---

ah, and that's what's being done elsewhere.